### DISCUSSION TITLE
Directive push in livewire

### DISCUSSION THREAD

**Discussion by Jesus-Rojas** [livewire/livewire#5431] [7. Bugs (archived)]
Hi, good day!

Excuse me, i found a bug in livewire. The directive push is working in a single level of livewire, example.

I have two components. The component A is inside component B, the directive push is working in component B but component A the directive is not working.

![image](https://user-images.githubusercontent.com/57029059/207049083-9945e753-543d-485f-b5f4-26841c431a0e.png)

**Comment by joshhanley** [comment]
@Jesus-Rojas yeah Livewire will not dynamically load scripts after page load, so you need to make sure that your child component that has `@push` in it, is being displayed during the initial page render the same as the parent component.

Hope this helps!

**Reply by Jesus-Rojas** [reply]
@joshhanley I understand. Thank's

**Reply by joshhanley** [reply]
@Jesus-Rojas no worries! 🙂
